packages/wallets/src/components/Base/Divider/__tests__/Divider.spec.tsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import Divider from '../Divider';
+
+describe('Divider', () => {
+    const mockProps = {
+        color: '#f2f3f4',
+        margin: '1rem',
+        variant: 'vertical',
+    } as typeof Divider.defaultProps;
```

Comment:
hey @mahdiyeh-deriv 
have one small suggestion here. 
___
As i know we should use `as` as last resort. I believe that we silently handle error here using assertion.
___
Another thing is that `defaultProps` type is `defaultProps?: Partial<P> | undefined`, for now thats fine because all props of `Divider` are optional. But if we add new **required** prop (e.g `size`) and will use `typeof <component>.defaultProps` our `size`  becomes optional, which is no true.
Long story short my proposal is to use `React.ComponentProps<typeof Divider>` type because it takes exactly reflects the same props of component and type mockProps as:
```suggestion
    const mockProps: React.ComponentProps<typeof Divider> = {
        color: '#f2f3f4',
        margin: '1rem',
        variant: 'vertical',
    } ;
```